package.json

```diff
@@ -1,17 +1,45 @@
 {
   "name": "@clerk/javascript",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
   "workspaces": {
     "packages": [
       "packages/*"
     ]
   },
-  "license": "MIT",
-  "version": "0.0.0",
-  "private": true,
-  "packageManager": "npm@8.5.0",
-  "engines": {
-    "node": ">=16.8.0",
-    "npm": ">=8.5.0"
+  "scripts": {
+    "build": "FORCE_COLOR=1 turbo build --concurrency=${TURBO_CONCURRENCY:-80%}",
+    "bundlewatch": "turbo bundlewatch",
+    "changeset": "npx changeset",
```

Comment:
If we're changing this file that much anyways, can we remove the `npx` in favor of just calling the scripts like I did here?
https://github.com/clerkinc/javascript/pull/1891#discussion_r1361591281

Then I won't need to deal that much with merge conflicts and the change might actually make more sense in this PR anyways